packages/fxa-auth-server/test/oauth/key-management-scripts.js

```diff
@@ -39,6 +38,8 @@ describe('the signing-key management scripts', function () {
             FXA_OPENID_NEWKEYFILE: newKeyFile,
             FXA_OPENID_OLDKEYFILE: oldKeyFile,
             NODE_ENV: 'dev',
+            TS_NODE_TRANSPILE_ONLY: true,
```

Comment:
Should we do this for all the tests? https://github.com/mozilla/fxa/blob/99385e89a555afa85aa8b4b9ec66b9c5f0251c73/packages/fxa-auth-server/scripts/test-local.sh#L13

Reply:
maybe?